tests/unit/test_adapter_yaml.py

```diff
@@ -670,3 +670,74 @@ def test_section_whitelines(self) -> None:
         result = fix_code(source, config)
 
         assert result == fixed_source
+
+    def test_section_whitelines_begin_no_explicit_start(self) -> None:
+        """Check that no whitelines are added at start of file when explicit start \
+            is not applied."""
+        source = dedent(
+            # pylint: disable=C0303
+            """\
```

Comment:
The `source` is too big to only test that there are no whitelines at the start of the file. Couldn't you just use the `begin_section` part?

Reply:
Yes, not really a good reason for it other than that I started by copy/pasting the above test 🙃 .